skills/dff_mint_skill/scenario/response_funcs.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import logging
+import re
+from os import getenv
+
+import common.dff.integration.context as int_ctx
+from common.robot import check_if_valid_robot_command
+from df_engine.core import Actor, Context
+
+
+LANGUAGE = getenv("LANGUAGE", "EN")
+ROS_FLASK_SERVER = getenv("ROS_FLASK_SERVER")
+
+logging.basicConfig(format="%(asctime)s - %(pathname)s - %(lineno)d - %(levelname)s - %(message)s", level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+def get_respond_funcs():
+    return {
```

Comment:
on it

Reply:
The problem comes from the difference between hardware mouse pointer and hardware mouse strokes. There is no problem with imitatink keyboard keypresses, as there seems to be no difference between "faking" a keypress and really pressing a key on a physical keyboard (probably because there is a virtual keyboard in windows). 

We can easily move the mouse pointer any way we want, the problems surface when you discover that in-game camera movements (in games that allow player to move their camera using the mouse) is not based on mouse pointer movement, but on mouse strokes. So in order to imitate player moving their camera, you must first have a mouse-like device that you can "intercept" (this process literally takes control of a physical mouse and renders it useless while it is being intercepted). When you intercept the mouse (for more details, look at [minecraft interface repo](https://github.com/deeppavlov/minecraft-interface/blob/0dd3451679acc176fe2f5b764f40522a0ea6b67e/game_minecraft.py#L113C5-L113C5)) you can imitate mouse strokes and move the camera the way the game wants you to.

But the interception code is really hard to read and to get a sliver of understanding the logic behind it, you would have to dive very deep into very low-level windows-specific concepts. So modifying the code is probably not the way to go, but I am looking into somehow creating a fake hardware mouse from a predefined config so that the end-user doesn't have to do weird stuff like moving their mouse in order for the script to intercept it.